src/qt/veil/veilstatusbar.cpp

```diff
@@ -40,6 +40,11 @@ void VeilStatusBar::updateSyncStatus(QString status){
     ui->btnSync->setText(status);
 }
 
+void VeilStatusBar::updateSyncIndicator(int height){
+    QString str;
+    ui->btnSyncIndicator->setText("Height: " + str.setNum(height));
```

Comment:
Text that the user can see should be wrapped with a translation method. We need to be proactive to any text changes made to the wallet. Example -> `tr("Height: ")`

Reply:
Hey @blondfrogs thanks, done & committed.